src/material/button/_button-base.scss

```diff
@@ -47,6 +47,11 @@
     z-index: 1;
   }
 
+  .mat-icon {
```

Comment:
We should be able to combine this with the `.mdc-button__label` right above (plus the `position: relative`).